README.rst

```diff
@@ -3,50 +3,53 @@
 frontend-app-ecommerce
 ======================
 
-This is a micro-frontend application responsible for the display of an authenticated user's order history. Please tag **@edx/revenue-squad** on any PRs or issues.
+Please tag **@edx/revenue-squad** on any PRs or issues.
 
-----------
+Introduction
+------------
 
-Development
------------
+This is a micro-frontend application responsible for the display of an authenticated user's order history.
 
-Start Devstack
-^^^^^^^^^^^^^^
+Installation
+------------
 
-To use this application `devstack <https://github.com/edx/devstack>`__ must be running and you must be logged into it.
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
 
--  Start devstack
--  Log in (http://localhost:18000/login)
-Start the development server
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Install Devstack using the `Getting Started <https://github.com/edx/devstack#getting-started>`_ instructions.
 
-In this project, install requirements and start the development server by running:
+2. Start up Devstack, if it's not already started.
 
-.. code:: bash
+3. Log in to Devstack (http://localhost:18000/login )
 
-   npm ci
-   npm start # The server will run on port 1996
+4. Within this project, install requirements and start the development server:
 
-Once the dev server is up visit http://localhost:1996.
+   .. code-block::
 
-Note: ``npm ci`` is recommended over ``npm install`` to match the way CI and production builds work and avoid unintentional changes to ``package_lock.json`` when doing other work.
```

Comment:
I believe we added this explanation somewhat recently due to the confusion of running `npm ci` vs. `npm install`

Reply:
Yeah, I'd leave this line, it seems educational

Reply:
I partially removed it because it was misleading... `npm install` doesn't result in changes in `package-lock.json` unless someone else forgot to check in an update to it, or is using a drastically different `npm` verison.  

That said, it's true `npm ci` it'll help you avoid unintentional changes, but may also be obscuring issues... 🤔